weave/mongo_loader.py

```diff
@@ -179,3 +179,70 @@ def load_mongo(self, uuids, **kwargs):
         self.load_mongo_metadata(uuids, collection=metadata_collection)
         self.load_mongo_manifest(uuids, collection=manifest_collection)
         self.load_mongo_supplement(uuids, collection=supplement_collection)
+
+
+    @staticmethod
+    def append_document (uuid : str, pantry, **kwargs):
```

Comment:
I don't think this function is needed. Can't we simply use load_mongo where ever this is called instead?

Reply:
We could do that. I made this function as a helper

Reply:
done